### DISCUSSION TITLE
posts/2023-02-07-school-immunology/

### DISCUSSION THREAD

**Discussion by giscus** [racheltho/rachel-blog#5] [Announcements]
# posts/2023-02-07-school-immunology/

Last year, I became captivated by a new topic in a way that I hadn’t felt since I first discovered machine learning

https://rachel.fast.ai/posts/2023-02-07-school-immunology/

**Comment by deepsatflow** [comment]
Hi Rachel,

The very first line describes my feelings too. In fact, I love it so much that I will repeat it here: "I like complexity, and I like challenges. When a new topic fascinates me, I want to devote all of my time to it." Anyway, it was nice going through the blog.

I also have a deep interest in biology. However, without a guided problem statement to pursue, recurring interest comes and vanishes, and I haven’t been able to devote as much time as I would hope. I do read books/new research out of curiosity from time to time, thinking someday I will work on something interesting. However, there is only so much you can read and keep the learning motivation aligned.

I would love to know if you are currently working on any interesting problems in the interaction of AI and immunology. Also, I would be interested in learning along with you. Please let me know.

Thanks,
Satish

**Reply by racheltho** [reply]
Hi Satish, thanks for reading and for the message.  I have found that some level of structure and a community of like-minded people are helpful for me in maintaining motivation and interest in a topic.  At times, I have had to create these myself (when I couldn't find an existing community or structure to plug into).

**Comment by nehal96** [comment]
Hi Rachel,

I came across this post at a great time (incidentally, from your immune cell network post) because I just quit my software engineering job to focus on learn biology and biotech full time. I too felt the overwhelming fascination you describe, and continue to feel it as I've dived deeper into the topic, mostly from books and articles. I particularly love Siddhartha Mukherjee's books -- the poetic prose and vivid accounts of the lives of scientists and their work, mixed with his own personal experiences, have made me fall in love with this interconnected web of subjects.

As someone who went through this journey, what did you find helpful at the very beginning? How did you pick between online courses? (Also, if you don't mind sharing, which online courses did you do?) I am considering grad school as well to formalize my learning but I'm not yet sure which of the many topics I wish to focus on (so, for now, I'm happy to continue exploring).

Excited to read your other posts and follow this AI x Immunology journey -- thanks for sharing!

-- Nehal

**Reply by racheltho** [reply]
Hi Nehal,
Thanks for reading. That is exciting that you are embarking on a career pivot as well!  I found using Anki flashcards helpful in my learning and have continued to practice them daily for the last 18 months, which I wrote about in another post.  For online classes, I will stop a class if I realize it's much too simple or too advanced for where I am, but otherwise I mostly try to stick with them, and I make flashcards as I go.  Some of the online classes I took are:
- EdX viruses course (on the simple side, so I ended up doing it together with my young daughter)
- Coursera 4 course series on immunology (overall, I thought this was really good, although could have used a bit more "big picture")
- HMX courses on immunology, genetics, and biochem (these are expensive, but I was happy with the quality)
I was partway through the above list of courses when I realized I wanted to go back to school formally, which is what I am focused on now.

Best wishes to you on your biology/biotech journey!

Take care,
Rachel
